README.md

```diff
@@ -8,51 +8,22 @@
 [![Swagger](https://img.shields.io/badge/-Swagger-%23Clojure?style=for-the-badge&logo=swagger&logoColor=white)](https://funding-service-design-account-store-dev.london.cloudapps.digital/#/default/core.account.post_account_by_email)
 ![Gunicorn](https://img.shields.io/badge/gunicorn-%298729.svg?style=for-the-badge&logo=gunicorn&logoColor=white)
 
-Repo for the funding service design account store.
+This repository is designed to handle the generating and providing of JSON payloads for accounts. It takes an email address and creates/returns a corresponding payload of account json
 
-Built with Flask + Connexion.
+[Developer setup guide](https://github.com/communitiesuk/funding-service-design-workflows/blob/main/readmes/python-repos-setup.md)
 
-Takes a email address and creates/returns a corresponding payload of account json.
 
-## Prerequisites
-- python ^= 3.10
+This service depends on:
+- A postgres database
+- funding-service-design-application-store
+- funding-service-design-assessment-store
```

Comment:
It doesn't depend on these services.... Just the database

Reply:
have removed